frame/asset-conversion/src/lib.rs

```diff
@@ -357,7 +357,7 @@ pub mod pallet {
 	}
 
 	#[pallet::hooks]
-	impl<T: Config> Hooks<T::BlockNumber> for Pallet<T> {
+	impl<T: Config> Hooks<frame_system::pallet_prelude::BlockNumberFor<T>> for Pallet<T> {
```

Comment:
Can you please do all these changes in a separate pr? No need to have them in here.

Reply:
@bkchr These are required once we remove `BlockNumber` from `frame_system::Config`. Do you suggest one PR to only add `Block` and the second one to remove `Header` and `BlockNumber`?